_rules/meta-viewport-b4f0c3.md

```diff
@@ -3,7 +3,7 @@ id: b4f0c3
 name: meta viewport does not prevent zoom
 rule_type: atomic
 description: |
-  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+  This rule checks that the `meta` element preserve the user agent ability to zoom.
```

Comment:
Suggest "retains" instead of "preserves"

Reply:
https://wikidiff.com/retain/preserve - I think "preserves" is better.

Alternatively:
name: meta viewport is not preventing zoom
description: This rule checks that the `meta` element is not preventing the user agent ability to zoom.

Reply:
Or...
name: meta viewport is preserving zoom
description: This rule checks that the `meta` element is preserving the user agent ability to zoom.

Reply:
Thank you for your comments. I'm not en english native speaker I can't have an opinion. @WilcoFiers @Jym77 @jkodu what is your opinion: retains vs preserves?

Reply:
I have no preference.